dashboard/lib/widgets/sign_in_button/sign_in_button_web.dart

```diff
@@ -0,0 +1,18 @@
+// Copyright 2019 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+
+import 'package:google_sign_in_platform_interface/google_sign_in_platform_interface.dart';
+import 'package:google_sign_in_web/google_sign_in_web.dart';
+
+/// Widget that users can click to initiate the Sign In process.
+class SignInButton extends StatelessWidget {
+  const SignInButton({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return (GoogleSignInPlatform.instance as GoogleSignInPlugin).renderButton();
```

Comment:
We can tweak the looks of the button, see a configurable demo here:

https://www.dit-gis-button-test.web.app